### DISCUSSION TITLE
Cookie Value is not decrypted

### DISCUSSION THREAD

**Discussion by iBiocide** [moonD4rk/HackBrowserData#444] [Q&A]
## Describe the bug
Output has most values empty for mostly Chrome browser

## Log Output
`{
    "Host": "www.enzahome.com.tr",
    "Path": "/",
    "KeyName": "sessionid",
    "Value": "",
    "IsSecure": true,
    "IsHTTPOnly": true,
    "HasExpire": true,
    "IsPersistent": true,
    "CreateDate": "2024-10-24T06:44:13.411077+03:30",
    "ExpireDate": "2024-10-24T09:44:13.411077+03:30"
  },
`
## Expected vs Actual Behavior
Describe what you expected to happen and what actually happened.

## Desktop (please complete the following information):
Select the operating system(s) you are using:
- [ ] Windows

- OS Version: 11
- OS Architecture: [64-bit]
- Browser Name: All
- Browser Version: Version 130.0.6723.70 (Official Build) (64-bit)

## Additional Details
- [ ] I ran `hack-browser-data` with administrator/root privileges.

## Checklist
- [ ] I have checked the [existing issues](https://github.com/moonD4rk/HackBrowserData/issues) for similar problems.

## Screenshots/Videos
If applicable, add screenshots or videos to help explain your problem.

## Additional context

i tried to manually decrypt the value myself usign payes.aes in python but i realized the problem is maybe about encoding, here is an example of encrypted value: 
![Capture](https://github.com/user-attachments/assets/dff99d47-8998-4f55-a269-d754b896e30a)

`v20r8‚iH‹¤šõ9AÚÎ|íi€)µ]ýZµœF!²I»™Zæ»h¡å[}àçâxdMòyÍl_îýiÝß7IŠ
`

**Comment by moonD4rk** [comment]
This is a known issue, Chrome has updated the encryption method to version 20. 

https://github.com/moonD4rk/HackBrowserData/issues/431
